src/components/layout.tsx

```diff
@@ -0,0 +1,16 @@
+import Footer from "./footer";
+import Header from "./header";
+
+type LayoutProps = {
+  children: React.ReactNode;
```

Comment:
Ja osobne preferujem importy = namiesto `React.ReactNode` si importnúť ReactNode a použiť ten

Reply:
Ale to už je len taká maličkosť a moja preferencia 😄

Reply:
import { ReactNode } from "react";

type LayoutProps = {
  children: ReactNode;
};

bude to aj moja preferencia